### PR TITLE
Fix 02475_bson_each_row_format flakiness (due to small parsing block)

### PR DESCRIPTION
**Pull request by azat**
CI: https://s3.amazonaws.com/clickhouse-test-reports/59170/c82050d1925439f0ede2b32acb5b1b8df4acae5d/stateless_tests__release_/run.log

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit fa2b0132a368d6a4e2d06d539911129b1db2236c with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59253/fa2b0132a368d6a4e2d06d539911129b1db2236c/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59253/fa2b0132a368d6a4e2d06d539911129b1db2236c/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: tests/queries/0_stateless/02475_bson_each_row_format.sh [hunk 1]
```diff
@@ -5,6 +5,9 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
+# Disable parallel parsing to avoid mixing output in case of small input block (--min_chunk_bytes_for_parallel_parsing)
+CLICKHOUSE_CLIENT+="--allow_repeated_settings --max_threads 1"
```
Comment (Avogar) [inline tests/queries/0_stateless/02475_bson_each_row_format.sh:9]: But parallel parsing should preserve the original order of rows. I think the reason is in parallel output from storages (`parallelize_output_from_storages`) when we had small blocks read in File storage. But `max_threads=1` will also help in this case (but let's change the comment as it's misleading)

Comment (azat) [reply tests/queries/0_stateless/02475_bson_each_row_format.sh]: >when we had small blocks read in File storage

Yep, indeed.

>but let's change the comment as it's misleading

I've adjusted the comment, how does it looks to you?

**Review by Avogar — APPROVED** [review]

**Comment by azat** [comment]
>Performance Comparison [1/4] — Errors while building the report.

- #59287

>Upgrade check (debug) — New settings are not reflected in settings changes history (see new_settings.txt)

- #59283

**Pull request merged**